resource_mock/__init__.py

```diff
@@ -70,20 +65,20 @@ def _previous(items, pivot):
         return result
 
     def _excel_reader(self) -> NoReturn:
+
         sites = [site for site in Site]
-        workbook = load_workbook(filename=os.path.join(self.path, '2018B-2019A Telescope Schedules.xlsx'))
+        workbook = load_workbook(filename=f'{self.path}/2018B-2019A Telescope Schedules.xlsx')
```

Comment:
Just out of curiosity, why did you change this? We really should not be creating paths with strings, but should always use `os.path.join`, as it handles slashes in a platform independent way and manages extra slashes properly.

Reply:
I didn't. Might got lost in the rebase or something.

Reply:
Yeah, `git rebase` has bit me in the ass more than a few times. 😒